src/java/com/unifina/security/SessionTokenAuthorizationHeader.java

```diff
@@ -0,0 +1,19 @@
+package com.unifina.security;
+
+public class SessionTokenAuthorizationHeader extends AuthorizationHeader {
+	private String token;
+
+	public SessionTokenAuthorizationHeader(String token){
+		this.token = token;
+	}
+
+	@Override
+	public String getHeaderName() {
+		return "Bearer";
+	}
+
+	@Override
+	public AuthenticationResult computeAuthenticationResult() {
+		return null;
```

Comment:
Still WIP, but tip: you can access Spring beans statically via
`SessionService sessionService = Holders.getApplicationContext().getBean(SessionService.class);`